Annotations/Classes/Extensions/NSBezierPathExtension.swift

```diff
@@ -49,130 +50,185 @@ extension NSBezierPath {
       case .closeSubpath:
         self.close()
       }
-      
-      pointsPtr.deinitialize()
-    }
-  }
-  
-  static func line(points: [CGPoint]) -> NSBezierPath {
-    let path = CGMutablePath()
-    for (index, point) in points.enumerated() {
-      if index == 0 {
-        path.move(to: point)
-      } else {
-        path.addLine(to: point)
-      }
+//      pointsPtr.deinitialize()
     }
-    let bezierPath = NSBezierPath(path: path)
-    return bezierPath
   }
-  
-  class func arrow(from start: CGPoint, to end: CGPoint, tailWidth: CGFloat, headWidth: CGFloat, headLength: CGFloat) -> NSBezierPath {
-    let length = hypot(end.x - start.x, end.y - start.y)
-    let tailLength = length - headLength
-    
-    func p(_ x: CGFloat, _ y: CGFloat) -> CGPoint { return CGPoint(x: x, y: y) }
-    let points: [CGPoint] = [
-      p(0, tailWidth / 2),
-      p(tailLength, tailWidth / 2),
-      p(tailLength, headWidth / 2),
-      p(length, 0),
-      p(tailLength, -headWidth / 2),
-      p(tailLength, -tailWidth / 2),
-      p(0, -tailWidth / 2)
-    ]
     
-    let cosine = (end.x - start.x) / length
-    let sine = (end.y - start.y) / length
-    let transform = CGAffineTransform(a: cosine, b: sine, c: -sine, d: cosine, tx: start.x, ty: start.y)
+    static func line(points: [CGPoint]) -> NSBezierPath {
+        let path = CGMutablePath()
+        for (index, point) in points.enumerated() {
+            if index == 0 {
+                path.move(to: point)
+            } else {
+                path.addLine(to: point)
+            }
+        }
+        let bezierPath = NSBezierPath(path: path)
+        return bezierPath
+    }
     
-    let path = CGMutablePath()
-    path.addLines(between: points, transform: transform)
-    path.closeSubpath()
+    static func rect(from start: CGPoint, to end: CGPoint, tailWidth: CGFloat, headWidth: CGFloat, headLength: CGFloat) -> NSBezierPath {
```

Comment:
@Vuong-Dao can you explain why we need tail and head widths for the rectangle? I would think they are only for arrows...